### PR TITLE
Add alt prop for img element.

### PR DESCRIPTION
**Pull request by petrewoo**

### CODE REVIEW

File: content/docs/introducing-jsx.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ const element = <div tabIndex="0"></div>;
 You may also use curly braces to embed a JavaScript expression in an attribute:
 
 ```js
-const element = <img src={user.avatarUrl}></img>;
+const element = <img src={user.avatarUrl} alt='introducing-jsx'></img>;
```
Comment (fanny) [inline content/docs/introducing-jsx.md]: The `alt` would be better, if it related with the img content, so:
```suggestion
const element = <img src={user.avatarUrl} alt='avatar-url'></img>;
```

File: content/docs/introducing-jsx.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ Don't put quotes around curly braces when embedding a JavaScript expression in a
 If a tag is empty, you may close it immediately with `/>`, like XML:
 
 ```js
-const element = <img src={user.avatarUrl} />;
+const element = <img src={user.avatarUrl} alt='introducing-jsx'/>;
```
Comment (fanny) [inline content/docs/introducing-jsx.md]:
```suggestion
const element = <img src={user.avatarUrl} alt='avatar-url'></img>;
```

File: content/docs/introducing-jsx.md (hunk 1 above)
Comment (petrewoo) [reply content/docs/introducing-jsx.md]: I re-read the doc and i think you are right. 
`alt`  attribute specifies an alternate text for an image.
I have tried VoiceOver and i think code `src={user.avatarUrl}` is clear enough for everyone, so i removed value of `alt` prop.

Comment (fanny) [reply content/docs/introducing-jsx.md]: Yeah, but the `alt` attribute is a alternative text, when image is not rendered, so it's still cool keep the attribute.

**Comment by BartoszKlonowski** [comment]
Hello @petrewoo, thank you for the contribution!
Unfortunately the part you modify in this PR no longer exists in the latest version of docs, so I will close this PR as outdated.
But please feel invited to create new PRs for open issues!

**Pull request closed without merging**